extra/mondoo-slack-incident-response.mql.yaml

```diff
@@ -0,0 +1,73 @@
+packs:
+  - uid: mondoo-incident-response-slack
+    name: Mondoo Slack Incident Response
+    is_public: true
+    docs:
+      desc: |
+        ### Overview
+
+        The Mondoo Slack Incident Response query pack gathers configuration data about Slack for investigation during a security incident.
+
+        ### Prerequisites
+
+        To run this query pack, you will need access to the Slack API. To get a token, you need to create an App for the Slack workspace 
+        and assign the appropriate permissions:
+
+        1. Sign in to [the Slack website](https://api.slack.com/apps/), and view "Your Apps"
+        2. Click "Create New App"
+        3. Select "From scratch"
+        4. Enter an "App Name" e.g. cnquery and select the workspace, then click "Create App"
+        5. In the section "Add features & functionality" click on "Permissions"
+        6. Scroll to "Scopes" and then "User Token Scopes"
+ 
+        Note: Bots are very limited with their access, therefore we need to set the user scopes
```

Comment:
Note: Bots are very limited in their access; therefore we need to set the user scopes